src/main/cpp/t03_points.cpp

```diff
@@ -23,10 +23,27 @@
 
 #include "t03_points.h"
 #include <iostream>
-
+#include <cmath>
+#include <map>
+#include <vector>
 
 using namespace std;
 
 int t03_points() {
-    
+int N,x,y;
+float distance;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'distance' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4586019)